src/ng/directive/input.js

```diff
@@ -457,6 +457,9 @@ var inputType = {
     * The model must always be a Date object, otherwise AngularJS will throw an error.
     * Invalid `Date` objects (dates whose `getTime()` is `NaN`) will be rendered as an empty string.
     *
+    * The value of the resulting Date object will be set to Thursday at 00:00:00 of the requested week,
+    * due to ISO-8601 week numbering standards.
```

Comment:
Looks good. Can you please also add a link to https://en.wikipedia.org/wiki/ISO_8601#Week_dates?